### PR TITLE
feat: first commit for vue-meta webpack loader

### PR DESCRIPTION
**Pull request by pimlie**
@Atinux @manniL @pi0 Some feedback would be appreciated about whether this is the way to go as meant in the rfc

### CODE REVIEW

File: packages/webpack-loader/src/loader.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  if (incomingQuery.type !== 'custom' || incomingQuery.blockType !== 'head') {
+    this.callback(null, source, map)
+  }
+
+  const vnodes = compile(`<head>${source}</head>`)
+
+  const [ metaInfo, hasBindings ] = vnodesToMetaData(vnodes.ast.children)
+
+  let content
+  if (hasBindings) {
```
Comment (Atinux) [inline packages/webpack-loader/src/loader.js]: Smart for performances

**Comment by Atinux** [comment]
Really great work on the loader @pimlie 

At first sight I don't see any issue regarding this loader.

@znck to you mind taking a quick look at it please? 🙏

File: packages/webpack-loader/src/loader.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      Component.options.computed = {
+        ...Component.options.computed,
+        metaInfo() {
+          return ${templify(metaInfo)}
+        }
+      }`
+  } else {
+    content = `
+      Component.options.data = {
+        ...Component.options.data,
```
Comment (znck) [inline packages/webpack-loader/src/loader.js]: I think `data` would be a function in most cases.

File: packages/webpack-loader/src/plugin.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+module.exports = class VueMetaLoaderPlugin {
+  apply(compiler) {
+    const rules = compiler.options.module.rules
```
Comment (manniL) [inline packages/webpack-loader/src/plugin.js]:
```suggestion
    const { rules } = compiler.options.module
```

**Pull request closed without merging**